src/core/CMakeLists.txt

```diff
@@ -774,9 +774,9 @@ target_link_libraries(inviwo-core
         inviwo::span
         inviwo::half
         inviwo::flags
+        inviwo::tracy
```

Comment:
Should be:
$<$<BOOL:${IVW_USE_TRACY}>:inviwo::tracy>

?

Reply:
No this needs to always be there. We need to add macros in the code. they will come from here. This way it will work even you don't have tracy installed